DigitalLearningSolutions.Web/Controllers/SuperAdmin/Users/AdminUserSetPasswordController.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.SuperAdmin.Users
+{
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.Services;
+    using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Users;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.FeatureManagement.Mvc;
+    using System.Threading.Tasks;
+
+    [FeatureGate(FeatureFlags.RefactoredSuperAdminInterface)]
+    [Authorize(Policy = CustomPolicies.UserSuperAdmin)]
+    [SetDlsSubApplication(nameof(DlsSubApplication.SuperAdmin))]
+    [SetSelectedTab(nameof(NavMenuTab.Admins))]
+    [Route("SuperAdmin/Users/AdminUserSetPassword/{userId:int}")]
+    public class AdminUserSetPasswordController : Controller
```

Comment:
Worried about the naming, here, @sarathlal-sarangadharan. "AdminUser" suggests that this is dealing with admin accounts rather than user accounts. Could we update refs to AdminUser to SuperAdminUser in this code to be more specific?